pkgs/by-name/sp/spacetimedb/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  perl,
+  git,
+}:
+rustPlatform.buildRustPackage rec {
```

You can use `finalAttrs` to make overriding easier

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

---

buildRustPackage now support `finalAttrs`? :o